shop_online_api/app/controllers/api/v1/orders_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::V1::OrdersController < ApplicationController
+  def index
+    @orders = Order.all
+    render json: @users, status: :ok
+  end
+
+  def show
+    @user = User.find(params[:id])
+    render json: @user
+  end
+
+  def create
+    confirm_token = request.headers['Access-token']
+    @user = User.find_by(confirm_token: confirm_token)
+    @order = Order.new(order_params)
+    @order.update(user_id: @user.id)
+    order_items = params[:orderItems]
+    if order_items && @order.save
+      order_items.each do |order_item|
+        OrderItem.create(price: order_item[:price], quantity: order_item[:quantity], item_id: order_item[:id], order_id: @order.id)
```

Comment:
- should use `Transactions` in Rails
- http://api.rubyonrails.org/classes/ActiveRecord/Transactions/ClassMethods.html